internal/proposals/logs/dynamic-backend-configuration/fluentbit-operator.md

```diff
@@ -0,0 +1,82 @@
+The interface will be specific to the logging use case always (fluent-bit opens up to metrics nowadays, still it does not seem to be as multi-purpose as the otel-collector). As kyma is planning to provide interface for configuration of tracing and monitoring as well, these configurations will always be different by design starting by the CRD apigroup already. **-> Strategic risk**
+* Another risk is the **indirection on top of fluent-bit** not being in control of the kyma project. Assuming fluent-bit requires a patch which needs to be activated by some Daemonset config, then the time till that option will be available get prolonged by additionally waiting to get the operator fixed and released. **-> Investment risk**
+* The **strictness of the CRD definitions** might become another challenge, if there is some bug in the translation of the CRD into the actual configuration (a very specific field of a plugin configuration has a wrong validation rule), a fix must be created in the community first and released. Having a free-form plugin definition with a non strict-validation (but a dry-run) might be way more flexible from a dependency perspective. Furthermore all standard features are not supported yet. Means, when new features to a filter are added, then these features have to be added to the FluentBit Operator as well, which is a downside relating to time. **-> Investment risk**
+* Furthermore, the FluentBit Operator **lacks important features** (secret injection, config check, bad config rollback, etc.), on which no concrete plans are published. Adding these features will require cooperation with the project and potential investments **-> Strategic risk**
+* The operator is too early (no stable release even), it is not obvious if it will find adoption and the project will have an active community long-term **-> Strategic risk**
+* An external component will bring maintenance efforts, the image must be hardened and often even duplicated to keep it clean from known vulnerabilities. Furthermore bugs must be fixed/workarounded on an external codebase. **-> Investment risk**
+* Having a custom operator to keep the independence in the user-facing interface and additional have the advantage of the fluent-bit operator as internal mechanism will increase the dependencies heavy, and adds another component to maintain from all kind of aspects like for example security scanning.
+
+## Chances custom operator
+* The custom operator will be written as part of the kyma team and will be in full control, especially it's interface can be designed so that it fits 100% into the usage scenarios. Furthermore, it must not be specific to the logging scenario, it can be extended to any other observability related usage and has the chance of providing an unified and consistent approach to the users.
```

Comment:
```suggestion
* The custom operator will be written as part of the kyma team and will be in full control, especially it's interface can be designed so that it fits 100% into the usage scenarios. Furthermore, it does not have to be specific to the logging scenario, it can be extended to any other observability related usage and has the chance of providing an unified and consistent approach to the users.
```